pkg/cloud/aws/scanner/scanner.go

```diff
@@ -44,11 +44,11 @@ func (s *AWSScanner) Scan(ctx context.Context, option flag.Options) (scan.Result
 	}
 
 	if option.Debug {
-		scannerOpts = append(scannerOpts, options.ScannerWithDebug(&defsecLogger{}))
+		scannerOpts = append(scannerOpts, options.ScannerWithDebug(&DebugLogger{}))
```

Comment:
How about passing a [named logger](https://pkg.go.dev/go.uber.org/zap#Logger.Named) as a field?   Zap will take care of adding the path to the logger name itself 

```suggestion
		scannerOpts = append(scannerOpts, options.ScannerWithDebug(&DebugLogger{log.Logger.Named("aws")}))
```

Reply:
`log.Logger.Named()` returns a sugaredlogger which doesn't satisfy the interface conditions. 

Instead I refactored the DebugLogger into the `log` package which would help us re-use it across other packages and added a name as a field as you mentioned.